### PR TITLE
fixed formatting and test_playbook [infinipoint-1014]

### PR DESCRIPTION
**Pull request by etInfp**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [x] In Progress
- [x] Ready
- [] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by hod-alpert** [comment]
@barchen1 Note that this PR should also fixes https://github.com/demisto/content/pull/8697

**Comment by barchen1** [comment]
@etInfp Please pull from demisto/master
Files like config.yml and Utils/release_notes_generator.py  should not appear in this PR.

File: Packs/Infinipoint/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Infinipoint 
+- Upgrade of test playbook
```
Comment (barchen1) [inline Packs/Infinipoint/ReleaseNotes/1_0_2.md:3]: No need to add here the changes in the test playbook
You need to write here only the things that user will see, for example
 - Updated the **infinipoint-run-queries** command name to **infinipoint-execute-action**.

File: Packs/Infinipoint/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- Fixed formatting 
+- Adding explanations
```
Comment (barchen1) [inline Packs/Infinipoint/ReleaseNotes/1_0_2.md:5]: Use our default format for this change:
- Documentation and metadata improvements.


Please take a look here https://xsoar.pan.dev/docs/integrations/release-notes and change the release notes accordingly.

File: Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml [hunk 3]
```diff
@@ -578,10 +578,10 @@ script:
     - contextPath: Infinipoint.Networks.Info.unmanagedCount
       description: Unmanaged devices count
       type: Number
-  - name: infinipoint-get-assets-hardware
-    description: get assets hardware
+  - name: infinipoint-get-assets-devices
```
[the remark this answers is not in the document]
Comment (etInfp) [reply Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml:581]: I run this again on my env and its look well 
image

File: Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml [hunk 4]
```diff
@@ -578,10 +578,10 @@ script:
...
+    description: Get managed devices
     arguments:
-    - name: host
+    - name: $host
```
[the remark this answers is not in the document]
Comment (etInfp) [reply Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml:584]: This is our parameters

File: Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml [hunk 5]
```diff
@@ -766,8 +766,8 @@ script:
     - contextPath: Infinipoint.Assets.User.username
       description: Username
       type: String
-  - name: infinipoint-get-action
-    description: get action
+  - name: infinipoint-get-action-results
```
[the remark this answers is not in the document]
Comment (etInfp) [reply Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml:769]: We know, we prefer to change it at such an early stage

File: Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml [hunk 6]
```diff
@@ -819,8 +819,8 @@ script:
     - contextPath: Infinipoint.Scripts.Search.osType
       description: OS type
       type: Number
-  - name: infinipoint-run-queries
-    description: run queries
+  - name: infinipoint-execute-action
```
[the remark this answers is not in the document]
Comment (etInfp) [reply Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml:822]: We know, we prefer to change it at such an early stage

**Review by barchen1 — APPROVED** [review]

**Pull request merged**